packages/react-component-library/src/components/Timeline/TimelineEvent.tsx

```diff
@@ -7,68 +7,71 @@ import { useTimelinePosition } from './hooks/useTimelinePosition'
 export interface TimelineEventWithRenderContentProps
   extends ComponentWithClass {
   children?: never
-  endDate: Date
-  render: () => React.ReactNode
   startDate: Date
-  status?: string
+  endDate: Date
```

Comment:
Is there a reason for reordering these props so they are no longer alphabetical?